### ISSUE TITLE
Narrator scan mode crashes terminal

### ISSUE THREAD

**Issue by benweedon** [microsoft/terminal#6402]
# Environment

Windows build number: Microsoft Windows [Version 10.0.18363.836]
Windows Terminal version (if applicable): 1.0.1401.0

Any other software?: Narrator, the version which comes with this windows build

# Steps to reproduce

* Start Narrator using Win+Ctrl+Enter
* Open the Windows Terminal app
  * Make sure the Narrator cursor (blue box) is on the terminal cursor, and/or that Narrator says "editing" as part of the information when you navigate to the terminal window (if Narrator is not focused on the correct place in the terminal window, alt-tabbing away and back should fix it).
* Activate scan mode with Caps+Space (Narrator will say "scan")
* Press the down arrow key (Narrator will say "title bar")
  * If Narrator does not announce "title bar", it may be further down in the UI tree. Keep pressing the down arrow until you reach it
* Press the up arrow key, and the Windows Terminal app crashes

# Expected behavior

I would expect the terminal app not to crash, and for Narrator focus to return to the edit box.

# Actual behavior

The terminal app crashes.

# Feedback hub link

https://aka.ms/AA8mlsm

**Comment by benweedon** [comment]
It's also worth noting that this doesn't repro using non-scan mode commands. Navigating away from the "title bar" element with Caps+left arrow instead of just up arrow does not crash the app.

Some other scan mode navigation commands seem to have the same effect. If at the "title bar" element you press left arrow instead of up arrow, you are returned to the edit pane (specifically a space in the bottom right corner of the edit pane). If you then press right arrow, however, the app crashes as well.
